src/GitTools.jl

```diff
@@ -78,6 +78,7 @@ function normalize_url(url::AbstractString)
 end
 
 function clone(url, source_path; header=nothing, kwargs...)
+    @assert !isdir(source_path) || isempty(readdir(source_path))
```

Comment:
I removed my `Pkg` dev dir by accident when cloning to `pwd()` so ...

Reply:
yeah, good idea